lightning/src/chain/package.rs

```diff
@@ -1297,10 +1301,19 @@ where
 				// ...else just increase the previous feerate by 25% (because that's a nice number)
 				let bumped_feerate = previous_feerate + (previous_feerate / 4);
 				let bumped_fee = bumped_feerate * predicted_weight / 1000;
+				log_trace!(logger, "Attempting forced 25% fee rate bump from {} s/kWU ({} s) to {} s/kWU ({} s)", previous_feerate, previous_fee, bumped_feerate, bumped_fee);
+
 				if input_amounts <= bumped_fee {
 					log_warn!(logger, "Can't 25% bump new claiming tx, amount {} is too small", input_amounts);
 					return None;
 				}
+
+				let remaining_output_amount = input_amounts - bumped_fee;
+				if remaining_output_amount < dust_limit_sats {
+					log_warn!(logger, "Can't new-estimation bump new claiming tx, output amount {} would end up below dust threshold {}", remaining_output_amount, dust_limit_sats);
```

Comment:
Nit: "new-estimation bump" seems confusing.

```suggestion
					log_warn!(logger, "Can't bump new claiming tx, output amount {} would end up below dust threshold {}", remaining_output_amount, dust_limit_sats);
```